Kind tests fail very often in CI

**Describe the bug**
It is very common to have at least one test failing for any given PR, because some antrea-agent instances take too long to become ready (even though they usually become ready).
```
2020-05-30T00:50:59.8019804Z === RUN   TestAntctlAgentLocalAccess
2020-05-30T00:52:30.8362226Z --- FAIL: TestAntctlAgentLocalAccess (91.03s)
2020-05-30T00:52:30.8366929Z ##[error]    fixtures.go:55: Creating K8s clientset
2020-05-30T00:52:30.8371399Z ##[error]    fixtures.go:61: Creating 'antrea-test' K8s Namespace
2020-05-30T00:52:30.8371960Z ##[error]    fixtures.go:38: Applying Antrea YAML
2020-05-30T00:52:30.8372418Z ##[error]    antctl_test.go:56: Error when setting up test: error when waiting for Antrea rollout to complete
2020-05-30T00:52:30.8372756Z === RUN   TestAntctlControllerRemoteAccess
2020-05-30T00:54:01.4683707Z --- FAIL: TestAntctlControllerRemoteAccess (90.64s)
2020-05-30T00:54:01.4769959Z ##[error]    fixtures.go:55: Creating K8s clientset
2020-05-30T00:54:01.4773347Z ##[error]    fixtures.go:61: Creating 'antrea-test' K8s Namespace
2020-05-30T00:54:01.4775580Z ##[error]    fixtures.go:38: Applying Antrea YAML
2020-05-30T00:54:01.4777435Z ##[error]    antctl_test.go:82: Error when setting up test: error when waiting for Antrea rollout to complete
2020-05-30T00:54:01.4778248Z === RUN   TestAntctlVerboseMode
2020-05-30T00:55:31.9269817Z --- FAIL: TestAntctlVerboseMode (90.46s)
2020-05-30T00:55:31.9288207Z ##[error]    fixtures.go:55: Creating K8s clientset
2020-05-30T00:55:31.9299942Z ##[error]    fixtures.go:61: Creating 'antrea-test' K8s Namespace
2020-05-30T00:55:31.9307996Z ##[error]    fixtures.go:38: Applying Antrea YAML
2020-05-30T00:55:31.9316889Z ##[error]    antctl_test.go:133: Error when setting up test: error when waiting for Antrea rollout to complete
```

I have been unable to reproduce it locally on my laptop or on Azure VMs that mimic the Github Linux worker VMs. However, I was able to ssh into a worker and inspect the agent logs:
```
I0605 02:29:16.274346       1 agent.go:50] Starting Antrea agent (version v0.5.0)
I0605 02:29:16.274411       1 client.go:33] No kubeconfig file was specified. Falling back to in-cluster config
I0605 02:29:16.275453       1 client.go:23] No antrea kubeconfig file was specified. Falling back to in-cluster config
I0605 02:29:16.275520       1 ovs_client.go:66] Connecting to OVSDB at address /run/openvswitch/db.sock
I0605 02:29:17.284701       1 agent.go:165] Setting up node network
E0605 02:34:24.042426       1 agent.go:445] Failed to get node from K8s with name kind-worker2: Get https://10.96.0.1:443/api/v1/nodes/kind-worker2: read tcp 172.17.0.2:59944->10.96.0.1:443: read: connection
timed out
F0605 02:34:24.042787       1 main.go:57] Error running agent: error initializing agent: Get https://10.96.0.1:443/api/v1/nodes/kind-worker2: read tcp 172.17.0.2:59944->10.96.0.1:443: read: connection timed out
```

I think this might be yet another case of stale conntrack mappings caused by the OVS bridge default behavior in standalone mode (`NORMAL` action). @tnqn do you think it's possible? I'm currently experimenting with creating the br-int bridge with `stand-alone=secure` to see if it solves the issue.

--- a/build/yamls/patches/kind/startAgent.yml
+++ b/build/yamls/patches/kind/startAgent.yml
@@ -1,0 +1,11 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: antrea-agent
+spec:
+  template:
+    spec:
+      containers:
+      - name: antrea-agent
+        command: ["/bin/sh"]
+        args: ["-c", "sleep 2; exec antrea-agent --config /etc/antrea/antrea-agent.conf"]

--- a/hack/generate-manifest.sh
+++ b/hack/generate-manifest.sh
@@ -235,9 +235,13 @@ if $KIND; then
 
     # add tun device to antrea OVS container
     $KUSTOMIZE edit add patch tunDevice.yml
-    # antrea-ovs should use start_ovs_netdev instead of start_ovs to ensure that the br_phy bridge
+    # antrea-ovs should use start_ovs_netdev instead of start_ovs to ensure that the br-phy bridge
     # is created.
     $KUSTOMIZE edit add patch startOvs.yml
+    # this adds a small delay before running the antrea-agent process, to give the antrea-ovs
+    # container enough time to set up the br-phy bridge.
+    # workaround for https://github.com/vmware-tanzu/antrea/issues/801
+    $KUSTOMIZE edit add patch startAgent.yml
     # change initContainer script and remove SYS_MODULE capability
     $KUSTOMIZE edit add patch installCni.yml
 

